pages/price-feeds/getting-started.mdx

```diff
@@ -0,0 +1,15 @@
+## Getting Started
+
+Integrating Pyth price feeds into your application is quick and easy.
```

Comment:
I think the text here should give them a menu of the different paths they can take through the document. I.e., it should point them to the build or learn section based on their use case. 

Example would be:
"Pyth offers several different resources to get started with price feeds. The Build section provides guides and resources for developers integrating with Pyth price feeds. The Learn section provides general material for anyone interested in understanding how the protocol works."

It's generally good practice for the first paragraph of any document to summarize the rest of the document.